graphql-server/pkg/models/validator.go

```diff
@@ -48,3 +60,53 @@ type ValidatorInfo struct {
 	ProposalVotes    []*ProposalVote    `bun:"rel:has-many,join:self_delegate_address=voter_address"`
```

Comment:
if we change these to []ProposalVote and []ProposalDeposit here, no need to do nil check below

we should review and remove unnecessary ptr types like these

Reply:
let's fix altogether in https://github.com/oursky/likedao/issues/229